### PR TITLE
Releasing v0.7.0

### PR DESCRIPTION
**Pull request by viktor-f**
**What this PR does / why we need it**: Releasing v0.7.0 please look through the commits and check that all changes seem to be present. The changes have been fixes as a part of the qa testing for this release. They will also be merged to main.

The other changes since v0.6.0 are not visible here since we recently created this repo and the old history is not available.

**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

### CODE REVIEW

**Review by danielharr — APPROVED** [review]
Great! :tada: 
LGTM

File: README.md [hunk 1]
```diff
@@ -65,7 +63,7 @@ The apps are installed using a combination of helm charts and manifests with the
 * [jq](https://github.com/stedolan/jq) (tested with jq-1.6)
 * [sops](https://github.com/mozilla/sops) (tested with 3.6.1)
 * [s3cmd](https://s3tools.org/s3cmd) available directly in ubuntus repositories (tested with 2.0.1)
-* [yq](https://github.com/mikefarah/yq) (tested with 3.3.2)
+* [yq](https://github.com/mikefarah/yq) (tested with 3.4.1)
```
Comment (OlleLarsson) [inline README.md:66]: Should we mention in the changelog that we have bumped the dependency on `yq`?

Comment (viktor-f) [reply README.md:66]: Yes, good point. Will fix

File: bin/init.bash [hunk 2]
```diff
@@ -196,6 +189,7 @@ set_elasticsearch_config() {
     esac
 
     replace_set_me "$1" 'elasticsearch.useRegionEndpoint' "$use_regionendpoint"
+    replace_set_me "$1" 'elasticsearch.snapshotRepository' "s3_${CK8S_CLOUD_PROVIDER}_7.x"
```
Comment (Xartos) [inline bin/init.bash:192]: Why do we need the cloud provider and version in this? Isn't this only the name of the repo?
```suggestion
    replace_set_me "$1" 'elasticsearch.snapshotRepository' "s3_repository"
```

**Pull request merged**